cloud/azure/app-service/variables.tf

```diff
@@ -106,7 +106,7 @@ variable "response_time_threshold_critical" {
   default     = 10
 }
 
-variable "response_time_threshold_warning" {
+variable "response_time_threshold_major" {
   description = "Warning threshold for response_time detector"
```

Comment:
It's not Warning threshold, but Major. You need replace all `Warning` by `Major`